models/clm/tutorial/README.rst

```diff
@@ -0,0 +1,1591 @@
+ 2022-01-14 14:21:33: model execution starting 
+ ---------------------------------------------------
+ 2022-01-14 14:23:58: model execution success 
+ ---------------------------------------------------
+ 2022-01-14 14:23:58: case.run success 
+ ---------------------------------------------------
+
+A failed run will provide an error message and a log file either from
+CESM or DART that hopefully provides more details of the error. This
+will look like this:
+
+::
+
+ ---------------------------------------------------
+ 2022-01-14 14:24:57: case.run starting 
+ ---------------------------------------------------
+ 2022-01-14 14:24:58: model execution starting 
+ ---------------------------------------------------
+ 2022-01-14 14:25:08: model execution error 
+ ERROR: Command: 'mpiexec_mpt -p "%g:"  -np 360  omplace -tm open64 
+ /glade/scratch/bmraczka/cesm2.2.0/clm5_SWE0_MissingVal/bld/cesm.exe 
+ >> cesm.log.$LID 2>&1 ' failed with error '' from dir 
+ '/glade/scratch/bmraczka/cesm2.2.0/clm5_SWE0_MissingVal/run'
+ ---------------------------------------------------
+ 2022-01-14 14:25:08: case.run error 
+ ERROR: RUN FAIL: Command 'mpiexec_mpt -p "%g:"  -np 360  omplace -tm open64
+ /glade/scratch/bmraczka/cesm2.2.0/clm5_SWE0_MissingVal/bld/cesm.exe   >> 
+ cesm.log.$LID 2>&1 ' failed See log file for details: 
+ /glade/scratch/bmraczka/cesm2.2.0/clm5_SWE0_MissingVal/run/cesm.log.2465146.chadmin1.ib0.cheyenne.ucar.edu.220114-142457
+
+If the case ran successfully proceed to the next step in the tutorial, **but if 
+the case did not run successfully** locate the log file details which describe
+the error and resolve the issue.  Contact dart@ucar.edu if necessary.
+
+
+Just because an assimilation ran successfully (without errors) does not mean
+it ran with good performance. A simple, first check after any assimilation is to 
+make sure:
+
+1) Observations have been  accepted
+2) The CLM posterior member values are updated from their prior values
+
+A quick way to confirm observation acceptance and the posteriors have been updated
+is through the ``obs_seq.final`` file located in your case run folder. Below we provide
+an example of a successful update (``clm_obs_seq.2011-01-02-00000.final``)  which is
+derived from the same leaf area observation in ``obs_seq.2011-01-02-00000`` as described
+in Step 7.  
+
+::
+
+ OBS            3
+   6.00864688253571
+   5.44649167346675
+   5.45489142211957
+   5.45808308572015
+   3.479253215076174E-002
+   3.469211455745640E-002
+   5.41512442472872
+   5.41843086313655
+   5.44649167346675
+   5.44970758027390
+   5.50877061923831
+   5.51180677764943
+   5.46367193547511
+   5.46683825691274
+   5.44039845768897
+   5.44363195062815
+   0.000000000000000E+000
+   0.000000000000000E+000
+           2           4          -1
+ obdef
+ loc3d
+     5.235987755982989         0.000000000000000        -888888.0000000000     -2
+ kind
+          23
+     0     149750
+  0.200000000000000
+
+::
```

Comment:
I think this would benefit from a note that the snipped below is explaining what all the entries are in obs_seq.final

Reply:
Completed.